### PR TITLE
docs(readme): getting started steps

### PR DESCRIPTION
**Pull request by arjun27**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,42 @@
 # 🎭 playwright-test
 
-> This package is not ready for use. If you want to run tests with playwright, checkout [jest-playwright](https://github.com/mmarkelov/jest-playwright) for Jest or [karma-playwright-launcher](https://github.com/JoelEinbinder/karma-playwright-launcher) for Karma.
+> 🚧 This project is under development. See [Test runner integrations](https://playwright.dev/#version=v1.4.1&path=docs%2Ftest-runners.md&q=) to use Jest or Mocha with Playwright.
```
Comment (mxschmitt) [inline README.md:3]: lets maybe use the non-version bound link?

File: README.md [hunk 2]
```diff
@@ -1,20 +1,42 @@
...
 
-A test runner for running tests with Playwright.
+Build an end-to-end test suite with Playwright.
+
+- **Parallelized execution** across multiple browsers.
+- **Annotate your tests** to mark your tests as `flaky`, `fixme`, `skip`, `slow` or `fail` and get better reporting.
+- **Shape your tests** with built-in and custom fixtures.
```
Comment (mxschmitt) [inline README.md:9]: Pytest is doing a great way of explaining fixtures. They even redirect to the Wikipedia entry which could be useful: https://en.wikipedia.org/wiki/Test_fixture#Software

**Review by mxschmitt — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (arjun27) [reply README.md:3]: sure 👍

File: README.md (hunk 2 above)
Comment (arjun27) [reply README.md:9]: Thanks! Going to follow up on this in a separate PR.

**Pull request merged**